stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/service/sync/PushMessageSyncHandler.kt

```diff
@@ -79,22 +69,16 @@ public class PushMessageSyncHandler(private val service: Service) {
         startForegroundExecution(notificationConfig.smallIcon)
 
         val cid: String = firebaseMessageParser.parse(message).let { "${it.channelType}:${it.channelId}" }
```

Comment:
The same as the above. We assume that clients might have `ChatClient` not initialized, which is not correct. This branch should never have in a real app.